packages/experimental/Menu/src/MenuList/useMenuList.ts

```diff
@@ -0,0 +1,49 @@
+import { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
+import React from 'react';
+import { useMenuContext } from '../context/menuContext';
+import { MenuListProps, MenuListState } from './MenuList.types';
+
+export const useMenuList = (_props: MenuListProps): MenuListState => {
+  const context = useMenuContext();
+
+  // MenuList v2 needs to be able to be standalone, but this is not in scope for v1
+  // Assuming that checked information will come from parent Menu
+  const isCheckedControlled = typeof context.checked !== 'undefined';
+  const [checked, onCheckedChange] = useMenuCheckedState(isCheckedControlled, context);
```

Comment:
Tracking the checked state is in MenuList instead of Menu to make it easier in the future for MenuList to be standalone.